Source/WebKit/Shared/gtk/ArgumentCodersGtk.serialization.in

```diff
@@ -30,3 +30,18 @@ class WebCore::SelectionData {
     RefPtr<WebCore::SharedBuffer> customData()
     bool canSmartReplace()
 }
+
+struct WebKit::GtkSettingsState {
+    std::optional<String> themeName;
+    std::optional<String> fontName;
+    std::optional<int> xftAntialias;
+    std::optional<int> xftHinting;
+    std::optional<String> xftHintStyle;
+    std::optional<String> xftRGBA;
+    std::optional<int> xftDPI;
+    std::optional<bool> cursorBlink;
+    std::optional<int> cursorBlinkTime;
+    std::optional<bool> primaryButtonWarpsSlider;
+    std::optional<bool> overlayScrolling;
+    std::optional<bool> enableAnimations;
```

Comment:
`enableAnimations` doesn't seem to be in the encode nor decoded methods, but it is in the GtkSettingsState struct. Not sure if it's a problem?

Reply:
I checked `git blame` and that member was added later, by @TingPing . I wonder if it was just an overlook?

Reply:
Would have been. Something these prevent :)